xonsh/imphooks.py

```diff
@@ -23,7 +23,7 @@
 def ENCODING_LINE():
     # this regex comes from PEP 263
     # https://www.python.org/dev/peps/pep-0263/#defining-the-encoding
-    return re.compile(b"^[ tv]*#.*?coding[:=][ t]*([-_.a-zA-Z0-9]+)")
+    return re.compile(b"^[ tf]*#.*?coding[:=][ t]*([-_.a-zA-Z0-9]+)")
```

Comment:
huh... should this be 
```suggestion
    return re.compile(b"^[ \t\f]*#.*?coding[:=][ \t]*([-_.a-zA-Z0-9]+)")
```
?

Reply:
You're correct, the tabs etc should be escaped for the regex to match them
I just assumed the original code was correct without the escaping, so didn't adjust it in this way:(